src/drafts/MarkdownEditor/index.ts

```diff
@@ -39,5 +39,5 @@ export type {MarkdownEditorProps} from './MarkdownEditor'
 export type {Emoji} from './suggestions/_useEmojiSuggestions'
 export type {Mentionable} from './suggestions/_useMentionSuggestions'
 export type {Reference} from './suggestions/_useReferenceSuggestions'
-export type {FileUploadResult} from './_useFileHandling'
+export type {FileUploadResult, FileType} from './_useFileHandling'
```

Comment:
I added `FileType` to the `MarkdownEditor` exports because it's useful when building the `acceptedFileTypes` prop.